cns/service/main.go

```diff
@@ -81,7 +81,8 @@ const (
 	name                              = "azure-cns"
 	pluginName                        = "azure-vnet"
 	endpointStoreName                 = "azure-endpoints"
-	endpointStoreLocation             = "/var/run/azure-cns/"
+	endpointStoreLocationLinux        = "/var/run/azure-cns/"
+	endpointStoreLocationWindows      = "/k/azurecns/"
```

Comment:
I used that path at first but store package showed error and couldn't create the endpoint file.

Reply:
then it needs to come from config instead of being hardcoded. we could have `config_linux.go` and `config_windows.go` to populate defaults if it is not set

Reply:
We also already made the AgentBaker changes according to k/azurecns directory since it was provided int he CNS Configmap for Windows. I simply applied the value form the configmap